lib/ClangImporter/ClangImporter.cpp

```diff
@@ -6480,9 +6474,35 @@ static bool hasRequiredValueTypeOperations(const clang::CXXRecordDecl *decl) {
       }))
     return false;
 
+  // TODO: this should probably check to make sure we actually have a copy ctor.
   return true;
```

Comment:
Well this is awful. Swift Compiler Sources need this to say incorrect. I will put up a revert commit after this and work through these issues separately. This patch needs to land today.